projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -332,16 +332,10 @@ private V1beta1CronJob loadConfigurableV1beta1CronjobTemplate() {
       return null;
     }
 
-    // Check whether the configurable datajob template file exists.
-    File datajobTemplateFile = new File(datajobTemplateFileLocation);
-    if (!datajobTemplateFile.isFile()) {
-      log.warn("Datajob template location '{}' is not a file.", datajobTemplateFileLocation);
-      return null;
-    }
-
     try {
       // Load the configurable datajob template file.
-      return loadV1beta1CronjobTemplate(datajobTemplateFile);
+      return loadV1beta1CronjobTemplate(
+          new ClassPathResource(datajobTemplateFileLocation).getFile());
```

Comment:
Why using ClassPathResource for loading the data job template file, in favour of FileSystemResource?

about [classpathresource](https://docs.spring.io/spring-framework/docs/3.2.x/spring-framework-reference/html/resources.html#resources-implementations):

> supports resolution as java.io.File if the class path resource resides in the file system, but not for classpath resources which reside in a jar and have not been expanded (by the servlet engine, or whatever the environment is) to the filesystem

Going into that direction, I wonder why using the File outdated paradigm in favour of the Path API introduced with v1.7 from 2011?